django/test/client.py

```diff
@@ -601,12 +601,9 @@ def login(self, **credentials):
             return False
 
     def force_login(self, user, backend=None):
-        if backend is None:
```

Comment:
Removing this code now requires us to pass a backend into ```force_login``` if multiple authentication backends are configured. Was that intentional? It doesn't seem like this requirement is in line with what ```force_login``` is meant to be. After updating to 1.10, I now have hundreds of calls to ```force_login``` in my unit tests that need updating. This is an easy mechanical update, but was this commit intended to be a test-code-breaking change in this way? If not, I'm happy to go file a bug...